arm_1_joint_controller:
  type: "position_controllers/JointPositionController"
  joint: "arm_1_joint"
  pid: {p: 10.0, i: 0.05, d: 1.0}

arm_2_joint_controller:
  type: "position_controllers/JointPositionController"
  joint: "arm_2_joint"
  pid: {p: 10.0, i: 0.05, d: 1.0}

joint_state_controller:
  type: "joint_state_controller/JointStateController"
  publish_rate: 50

wheel_1_joint_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: "wheel_1_joint"
  pid: {p: 10.0, i: 0.1, d: 1.0}

wheel_2_joint_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: "wheel_2_joint"
  pid: {p: 10.0, i: 0.1, d: 1.0}

wheel_3_joint_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: "wheel_3_joint"
  pid: {p: 10.0, i: 0.1, d: 1.0}

wheel_4_joint_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: "wheel_4_joint"
  pid: {p: 10.0, i: 0.1, d: 1.0}